python/ray/worker.py

```diff
@@ -1681,6 +1696,8 @@ def connect(info, object_id_seed=None, mode=WORKER_MODE, worker=global_worker,
         driver_info["name"] = (main.__file__ if hasattr(main, "__file__")
                                else "INTERACTIVE MODE")
         worker.redis_client.hmset(b"Drivers:" + worker.worker_id, driver_info)
+        if not worker.redis_client.exists("webui"):
+            worker.redis_client.hmset("webui", {"url": info["webui_url"]})
```

Comment:
Should we use `set` or do you think we'll want to add more fields down the road?